Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,179 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
```

Comment:
Try not to commit more than is relevant for this particular PR, you didn't need to include the week1 homework in the week3 PR.